specs/templates/4-api-schemas.yaml

```diff
@@ -633,9 +623,9 @@ schemas:
             description: The maximum number of collateral inputs allowed in a transaction
             example: 3
             nullable: true
-          coins_per_utxo_word:
+          coins_per_utxo_size:
```

Comment:
Note: On mainnet, this is still `coins_per_utxo_word` until switching to db-sync v13.

Reply:
Ye, mainnet ones are will have heaps of more data inconsistencies , but that's OK since they match at 1.0.1 version